src/lib.rs

```diff
@@ -281,15 +281,16 @@ fn page_completions<C: Candidate, H: Helper>(
                 && cmd != Cmd::SelfInsert(1, ' ')
                 && cmd != Cmd::Kill(Movement::BackwardChar(1))
                 && cmd != Cmd::AcceptLine
-                && cmd != Cmd::AcceptOrInsertLine
+                && cmd != Cmd::Newline
+                && matches!(cmd, Cmd::AcceptOrInsertLine { .. })
```

Comment:
`&& !matches!(cmd, Cmd::AcceptOrInsertLine { .. })` (not matches)

Reply:
Fixed